src/transformers/models/table_transformer/modeling_table_transformer.py

```diff
@@ -814,15 +814,15 @@ def _set_gradient_checkpointing(self, module, value=False):
 
             Pixel values can be obtained using [`DetrImageProcessor`]. See [`DetrImageProcessor.__call__`] for details.
 
-        pixel_mask (`torch.LongTensor` of shape `(batch_size, height, width)`, *optional*):
+        pixel_mask (`torch.FloatTensor` of shape `(batch_size, height, width)`, *optional*):
```

Comment:
Here we are facing the same issue as in [here](https://github.com/huggingface/transformers/pull/25557#discussion_r1300015531) I've changed the docstring as the model seems to work with both types. In the case of the `decoder_attention_mask`, that is not used by the method so one can pass whatever she wants.

Reply:
Yep, I think `FloatTensor` is correct here!